docs/developer-guide/building-modules.md

```diff
@@ -38,11 +44,9 @@ Node.js can be installed manually or via package management, and often ships pre
 <output>v0.12.2</output>
 </pre>
 
-If you get an error, or the number you get does not match the most recent release shown on the [Node website](http://nodejs.org/), you need to install/upgrade Node.  If you're installing it on a personal machine, go to the Node website and click the Install button to download the installer suitable for your system:
+If you get an error, or the number you get does not match the most recent release shown on the [Node website](http://nodejs.org/), you need to install/upgrade Node.  
 
-* [Download the Node installer](http://nodejs.org/)
```

Comment:
I suggested to just leave one install method to keep things simple, and the other one works for all platforms. The method that's left includes the removed method